docker-compose-rule-junit4/build.gradle

```diff
@@ -7,10 +7,13 @@ testSets {
 build.dependsOn integrationTest
 
 dependencies {
-    compile project(':docker-compose-rule-core')
-    compile 'junit:junit'
+    api project(':docker-compose-rule-events-api:docker-compose-rule-events-api-objects')
+    api project(':docker-compose-rule-core')
 
-    testCompile 'org.hamcrest:hamcrest-core'
-    testCompile 'org.mockito:mockito-core'
-    testCompile 'org.assertj:assertj-core'
+    implementation 'com.google.guava:guava'
+    implementation 'junit:junit'
```

Comment:
I think this needs to be an API dependency because `DockerComposeRule` is a JUnit rule.